# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../wavemon/gcc34.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- wavemon-0.4.0b/conf.c.orig	2004-05-11 14:00:22.433704840 +0200
+++ wavemon-0.4.0b/conf.c	2004-05-11 14:01:33.102961480 +0200
@@ -411,6 +411,7 @@
 									break;
 					case t_sep: 	/* gcc -Wall sucks */
 					case t_func:
+					; /* and dummy coders, too */
 				}
 			}
 		}
@@ -482,6 +483,7 @@
 				case t_listval: sprintf(rv, "%s", (char *)ci->v);
 				case t_sep: break;
 				case t_func:
+				;
 			}
 			
 			add = 1;
--- wavemon-0.4.0b/conf_scr.c.orig	2004-05-11 14:01:54.314736800 +0200
+++ wavemon-0.4.0b/conf_scr.c	2004-05-11 14:02:28.469544480 +0200
@@ -64,6 +64,7 @@
 							break;
 			case t_sep:		/* gcc -Wall sucks */
 			case t_func:
+			; /* and dummy coders, too */
 		}
 
 		if (!item->unit) {
@@ -137,6 +138,7 @@
 						break;
 		case t_sep:		/* gcc -Wall sucks */
 		case t_func:
+		; /* and dummy coders, too */
 	}
 }	
 
--- wavemon-0.4.0b/wavemon.c.orig	2004-05-11 14:02:49.263383336 +0200
+++ wavemon-0.4.0b/wavemon.c	2004-05-11 14:03:31.590948568 +0200
@@ -170,7 +170,7 @@
 
 	if (conf.dump == 1) {
 	  dump_parameters(&conf);
-	  exit();
+	  exit(0);
 	}
 
 	/* initialize the ncurses interface */
